### PR TITLE
Bugfix MTE-4201 Keep latest Bitrise stack

### PR DESCRIPTION
**Pull request by clarmso**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/MTE-4201)

## :bulb: Description

We have used the latest Xcode, but Github Actions still bugs us to downgrade:
https://github.com/mozilla-mobile/firefox-ios/pull/23579

If we are using a newer version, we can discard such PRs. Let's refrain from create these PRs to reduce noise.

## :pencil: Checklist
You have to check all boxes before merging
- [ ] Filled in the above information (tickets numbers and description of your work)
- [ ] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed, I updated documentation / comments for complex code and public methods
- [ ] If needed, added a backport comment (example `@Mergifyio backport release/v120`)

### CODE REVIEW

File: test-fixtures/update.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ def default_stack():
     4. modify bitrise.yml (update stack value)
     '''
 
-    largest_semver = default_stack().split(pattern)[1]
+    default_semver = default_stack().split(pattern)[1]
```
Comment (clarmso) [inline test-fixtures/update.py:61]: Would it be more useful for us to query the latest non-edge version of Bitrise stack instead?

**Review by mdotb-moz — APPROVED** [review]
LGTM

File: test-fixtures/update.py (hunk 1 above)
Comment (isabelrios) [reply test-fixtures/update.py:61]: why?

**Review by isabelrios — APPROVED** [review]

File: test-fixtures/update.py (hunk 1 above)
Comment (clarmso) [reply test-fixtures/update.py:61]: If we query the latest dot release, we can be notified when the dot release become available.

Comment (clarmso) [reply test-fixtures/update.py:61]: 😅 For now, we can just tell Github Actions not to open a PR to roll back the stack to an older version.

**Pull request merged**